### ISSUE TITLE
Birghtness setting

### ISSUE THREAD

**Issue by DeeJee** [gadjet/1602-OLED-Arduino-Library#1]
Hello,

I am making use of your library to control my wide.hk 2002 OLED display. Your library sure makes using the thing easier. But I'd like to be able to  change the brightness. You you know if that it at all possible? 

Cheers,
Dennis

**Comment by gadjet** [comment]
Dennis,
It's been quite while since I looked at this library, off the top of my head I didn't implement any brightness control, I'll dig out the documentation and have a look to see if there is anything about setting the brightness.

**Comment by DeeJee** [comment]
Thanks. There is something called 'contrast' in the library which is supposed to set the brightness from completely off to maximum output. But when I set the contrast to 0x00 the display became a bit dimmer but it didn't turn completely off, which is what I expected.

**Comment by gadjet** [comment]
Yes there is a contrast adjustment but it made very little difference.....
